helm/cluster-api-provider-cloud-director-app/templates/crds/vcdclusters.yaml

```diff
@@ -0,0 +1,357 @@
+{{- if .Values.crds.create }}
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/capvcd-serving-cert
+    controller-gen.kubebuilder.io/version: v0.4.1
+  labels:
+    cluster.x-k8s.io/provider: infrastructure-vcd
+    cluster.x-k8s.io/v1alpha4: v1alpha4
+    cluster.x-k8s.io/v1beta1: v1beta1
+    clusterctl.cluster.x-k8s.io: ""
+    clusterctl.cluster.x-k8s.io/move: ""
+    {{- include "labels.common" . | nindent 4 }}

```

Comment:
i highly assume, that the current state only work if this chart got applied directly via `helm` (or `helm template`) when the flag `--include-crds` is set. GS app operator doesn't do it that way and therefore some hacks are required to get `CRDs` managed by the app operator:
* `crds` must be stored in a folder where name is not `crds` (e.g.: https://github.com/giantswarm/cluster-api-provider-openstack-app/tree/master/helm/cluster-api-provider-openstack/files)
* a `crd-install` job (plain `kubernetes job`) is needed (e.g.: https://github.com/giantswarm/cluster-api-provider-openstack-app/tree/master/helm/cluster-api-provider-openstack/templates/crd-install)

Reply:
I used this method because it's what I was comfortable working with by myself within a short timeframe but I definitely agree that the kustomize approach is what we should be using in the long run here.

I did not have to use the flag `--include-crds` to make it work. 

Moving crds to match capo.